src/runtime-rs/src/shim-proto/src/events_task.rs

```diff
@@ -0,0 +1,2702 @@
+// This file is generated by rust-protobuf 2.25.2. Do not edit
```

Comment:
I am not sure. We can put auto-generated code into `auto-generated/` directory, but rust can not include all .rs files under a directory automatically.  The `lib.rs` or `mod.rs`  is still to be modified manually. (Surely we can use `build.rs` or macros to make modification automatically, but I think it will be a bit more complicated)